### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by alganthe**
**When merged this pull request will:**
- Add french translation for:
- Advanced Fatigue
- Rearm
- Finger
- Nightvision
- Pylons
- mk6mortar
- cookoff

Translation notes:
shutter = obturateur.
cookoff was kept as is because I could not find a french equivalent.
Same thing for loadout, no accurate translation (technically loadout isn't even a word), I kept the one used in ace arsenal -> équipements.
some of the pylon settings were insanely long, so I had to shorten them using brackets ().

### CODE REVIEW

File: addons/nightvision/stringtable.xml [hunk 1]
```diff
@@ -252,6 +253,7 @@
             <German>Nebel in Nachtsicht</German>
             <Chinese>夜視鏡霧氣程度</Chinese>
             <Chinesesimp>夜视镜雾气程度</Chinesesimp>
+            <French>Ampleur du brouillard (JVN)</French>
```
Comment (ArwynFr) [inline addons/nightvision/stringtable.xml]:
```suggestion
            <French>Épaisseur du brouillard (JVN)</French>
```

File: addons/nightvision/stringtable.xml [hunk 2]
```diff
@@ -268,6 +271,7 @@
             <German>Nachtsichteffekte</German>
             <Chinese>夜視鏡效果程度</Chinese>
             <Chinesesimp>夜视镜效果程度</Chinesesimp>
+            <French>Ampleur des effets</French>
```
Comment (ArwynFr) [inline addons/nightvision/stringtable.xml]:
```suggestion
            <French>Intensité des effets</French>
```

File: addons/nightvision/stringtable.xml [hunk 3]
```diff
@@ -276,39 +280,45 @@
...
             <Chinesesimp>瞄准具模糊程度</Chinesesimp>
+            <French>Flou de visée</French>
         </Key>
         <Key ID="STR_ACE_NightVision_noiseScaling_DisplayName">
             <English>NVG Noise Scale</English>
             <German>Intensität des Bildrauschens</German>
             <Chinese>夜視鏡雜訊程度</Chinese>
             <Chinesesimp>夜视镜杂讯程度</Chinesesimp>
             <Japanese>暗視装置のノイズ度</Japanese>
+            <French>Ampleur du bruit (JVN)</French>
```
Comment (ArwynFr) [inline addons/nightvision/stringtable.xml]:
```suggestion
            <French>Intensité du bruit (JVN)</French>
```

File: addons/pylons/stringtable.xml [hunk 4]
```diff
@@ -159,6 +177,7 @@
             <Chinesesimp>导弹挂架整装所需时间(个别)</Chinesesimp>
             <Korean>파일런당 시간</Korean>
             <German>Zeit pro Außenlaststation</German>
+            <French>Temps par pylône</French>
```
Comment (ArwynFr) [inline addons/pylons/stringtable.xml]:
```suggestion
            <French>Durée par pylône</French>
```

File: addons/advanced_fatigue/stringtable.xml [hunk 5]
```diff
@@ -116,13 +116,15 @@
             <Chinesesimp>抖动因数</Chinesesimp>
             <Chinese>抖動因素</Chinese>
             <French>Facteur de stabilisation</French>
+            <Italian>Fattore di oscillazione</Italian>
```
Comment (dedmen) [inline addons/advanced_fatigue/stringtable.xml]: >French translation

:thinking:

Comment (alganthe) [reply addons/advanced_fatigue/stringtable.xml]: i'll check but it's likely another PR deleted that line before I PR'd this.

Comment (alganthe) [reply addons/advanced_fatigue/stringtable.xml]: it's due to how the merge was done @dedmen check the merge commit.

**Pull request merged**